desktop/src/gui.rs

```diff
@@ -193,6 +195,13 @@ impl RuffleGui {
         player.set_volume(self.volume_controls.get_volume());
     }
 
+    fn is_taking_screenshot(&mut self) -> bool {
```

Comment:
The name `is_taking_screenshot` sounds like this method doesn't change the state- could this just return `self.taking_screenshot`? (I think this approach would require adding a `set_taking_screenshot` function)

Reply:
Updated to use `set_taking_screenshot(bool state)` and `get_taking_screenshot() -> bool`